editoast/src/views/train_schedule/mod.rs

```diff
@@ -99,13 +131,23 @@ async fn delete(db_pool: Data<DbPool>, train_schedule_id: Path<i64>) -> Result<H
 
     Ok(HttpResponse::NoContent().finish())
 }
-#[derive(Debug, Deserialize)]
-pub struct BatchDeletionRequest {
+
+#[derive(Debug, Deserialize, ToSchema)]
+struct BatchDeletionRequest {
     ids: Vec<i64>,
 }
-//Delete multiple train schedule
+
+/// Delete multiple train schedules at once
+#[utoipa::path(
+    tag = "train_schedule",
+    // tag = "timetable",
```

Comment:
If we leave it as a comment, maybe be more explicit that it should also be `timetable` but that utoipa doesn’t allow it